apps/passport-client/src/pcdPackages.ts

```diff
@@ -78,6 +79,7 @@ async function loadPackages(): Promise<PCDPackage[]> {
     RSAImagePCDPackage,
     EmailPCDPackage,
     MessagePCDPackage,
-    PODPCDPackage
+    PODPCDPackage,
```

Comment:
A suggestion on the build issue CI is showing here.  I tried replacing the references to PCDPackage (with no tempalte args) in this file with this type:

    type AnyPCDPackage = PCDPackage<any, any, any, any, any>;

That seems to build successfully, though I haven't tested beyond that.  I think what's going wrong is that the default for the new template arg ends up setting it to `PCD<any, any>` which is less flexible than `any`.  You could also get around that if the arg had no default, which would force every PCD package to set it explicitly.

Notes in case this needs to go further: I think the use of `any` here (which is implicit when the template args are missing) effectively turns off type-checking.  Without that I think there wouldn't be any one type which would work perfectly for this object-containing-functions approach.  The default covariance of functions would work for return types here, but would be reversed for function args.  The canonical solution to that would be to move functions like verify() to be methods of a class, so that the arg is `this` and implicitly varies in type.  Or else functions like verify() could take an arg of type `PCD<Proof, Claim>` and do an `instanceof` check.  It is technically possible for someone to call these functions with the wrong type anyway (e.g. if calling deserialize on the wrong package for the type), but that's the nature of the looseness of TypeScript typing.